bgpd/bgp_evpn_mh.c

```diff
@@ -3460,13 +3460,10 @@ static void bgp_evpn_es_evi_vtep_re_eval_active(struct bgp *bgp,
 			   new_active ? "active" : "inactive");
 
 	/* add VTEP to parent es */
-	if (new_active) {
-		struct bgp_evpn_es_vtep *es_vtep;
-
-		es_vtep = bgp_evpn_es_vtep_add(bgp, evi_vtep->es_evi->es,
```

Comment:
I believe this is useful when debugging (gdb for instance)?

Reply:
What magic? I don’t know…